workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/NextSteps/NextSteps.tsx

```diff
@@ -48,6 +48,9 @@ type TypeNextStepBody = {
 	[key in string]?: ReactNode;
 };
 
+const CLOUD_APP_ERC = 'CLOUDAPP';
+const DXP_APP_ERC = 'DXPAPP';
```

Comment:
Such constants already exists in `ORDER_TYPES`

`/liferay-marketplace-custom-element/src/enums/Order.ts`

```suggestion
```